README.md

```diff
@@ -1,45 +1,38 @@
-# jawn-streamz
+# jawn-fs2
 
-Asynchronously parse [scalaz-streams](http4s://github.com/scalaz/scalaz-stream)
+Asynchronously parse [fs2](https://github.com/functional-streams-for-scala/fs2) streams
 to JSON values with [jawn](https://github.com/non/jawn).
 
 ## Example
 
 `sbt test:run` to see it in action:
 
 ```Scala
-import jawnstreamz._
+import jawnfs2._
 
 object Example extends App {
   // Pick your favorite supported AST (e.g., json4s, argonaut, etc.)
-  implicit val facade = jawn.ast.JawnFacade
-  // Read up to 64 bytes at a time
-  val chunkSizes: Process[Task, Int] = emitAll(Stream.continually(nextInt(64)))
+  implicit val facacde = jawn.ast.JawnFacade
   // From JSON on disk
-  val jsonSource = chunkSizes.through(io.chunkR(getClass.getResourceAsStream("/jawnstreamz/random.json")))
+  val jsonStream = io.file.readAll[Task](Paths.get("testdata/random.json"), 64)
   // Introduce up to a second of lag between chunks
-  val laggedSource = jsonSource.zipWith(awakeEvery(nextInt(1000).millis))((chunk, _) => chunk)
+  val laggedStream = jsonStream.chunks.zipWith(time.awakeEvery[Task](nextInt(1000).millis))((chunk, _) => chunk)
   // Print each element of the JSON array as we read it
-  val json = laggedSource.unwrapJsonArray.map(_.toString()).to(io.stdOutLines)
-  // First run converts process into a Task, second run executes the task for its effects
-  json.run.run
+  val json = laggedStream.unwrapJsonArray.map(_.toString).intersperse("\n").through(text.utf8Encode)
+  // run converts the stream into a Task, unsafeRun executes the task for its effects
+  json.to(io.stdout).run.unsafeRun
 }
 ```
 
-## Add jawn-streamz to your project
+## Add jawn-fs2 to your project
 
 Add to your build.sbt:
 
 ```
 resolvers += "bintray/rossabaker" at "http://dl.bintray.com/rossabaker/maven"
 
-libraryDependencies += "org.http4s" %% "jawn-streamz" % "0.8.1"
+libraryDependencies += "org.http4s" %% "jawn-fs2" % "0.9.0"
 
 // Pick your AST: https://github.com/non/jawn#supporting-external-asts-with-jawn
-libraryDependencies += "org.jsawn" %% "jawn-ast" % "0.8.4"
+libraryDependencies += "org.jsawn" %% "jawn-ast" % "0.10.1"
 ```
```

Comment:
I might leave the compatibility section as a historical note, and add `fs2-0.9 -> jawn-fs2-0.9`.